source/Calamari.Shared/Integration/Certificates/WindowsX509CertificateStore.cs

```diff
@@ -331,14 +314,14 @@ static void AddCertificateToStore(CertificateSystemStoreLocation storeLocation,
 
                         if (error == (int) CapiErrorCode.CRYPT_E_EXISTS)
                         {
-                            Log.Info($"Certificate '{subjectName}' already exists in store '{storeName}'.");
+                            log.Info($"Certificate '{subjectName}' already exists in store '{storeName}'.");
                             return;
                         }
 
                         throw new CryptographicException(error);
                     }
 
-                    Log.Info($"Imported certificate '{subjectName}' into store '{storeName}'");
```

Comment:
The use of these static `Log` methods made testing using the in-process Calamari fail (as opposed to many tests which actually invoke `Calamari.exe` and make debugging impossible)